docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+# Cellxgene Curation Tutorial
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png) --->
+
+<img align="right" src="https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png">
+
+
+<!--- ## Overview --->
+<!--- purpose: Motivate submission to the data portal give context --->
+
+[Cellxgene's publishing platform](https://cellxgene.cziscience.com/) and [interactive single cell data explorer](https://github.com/chanzuckerberg/cellxgene) is a system which is optimized for access, exploration and reuse of single cell data. In order to achieve these goals, the cellxgene platform currently accepts curated [AnnData](https://anndata.readthedocs.io/en/latest/#) objects adhering to a [succinct data schema](corpora_schema.md). Adherence to a standardized data schema allows for efficient navigation and integration of the growing number of single cell datasets that are becoming available. In this tutorial, you will learn the essential information for curating a single cell dataset using CZI's curation tools and uploading to the data portal. Hosting your data on the cellxgene portal will offer the following benefits:
+ - link permanence (you can reference in your publication without ever worrying about dead links)
```

Comment:
What can I reference in my publication?

Reply:
We should add a suggested sentence to be added to papers. It's frequently not highlighted that the data can be downloaded from the portal

Reply:
@brianraymor what about 

"link permanence (you can reference the public cellxgene Collection link your publication)"


@pablo-gar do you have suggested wording for sentence and location in tutorial?

Reply:
"Processed single-cell data can be downloaded and interactively browsed at [insert cellxgene collection link]"

Maybe somewhere in the "upload" section. I don't have a strong suggestion for the location.